app/javascript/pages/howToApply/how-to-apply.tsx

```diff
@@ -37,7 +37,8 @@ interface HowToApplyProps {
 }
 
 const submissionUrl = (listingId: string) => {
-  return `https://sfmoh.tfaforms.net/20?ListingID=${listingId}`
+  const formAssemblyUrl = process.env.FCFS_FORMASSEMBLY_URL || "https://sfmoh.tfaforms.net/20"
```

Comment:
What would be the best way to handle blank values? A default link like this, or raising an error?
@tallulahkay @alulabeshue-sfgov

Reply:
I think this is a design/product question! Can you ask bridget?

I would go default link, but idk how bad it would be if someone accidentally was applying with the wrong form

Reply:
An error would surface the issue quicker, especially if its caught by monitoring and there is an alert. Using the default could result in applicants applying using the wrong form and they would never be considered or we would catch it and would need to migrate the applications.

Reply:
Per designer instructions, a missing URL will hide the submit button.